### ISSUE TITLE
Install bed levelling sensor on alex-ender3

### ISSUE THREAD

**Issue by MatthewCroughan** [DoESLiverpool/somebody-should#1394]
@ajlennon has ordered 2 geeetech bltouch-like devices that are essentially a type of hall effect sensor. They are relatively accurate, good enough for what is required for FDM printing. I plan to install this as well on `alex-cr10s`. `matt-ender3` already has one installed.

**Comment by ajlennon** [comment]
Yeah they arrived a while ago. Should be by the machines?

**Comment by MatthewCroughan** [comment]
I've wrapped it up neatly but am not going to go through the same trouble I went through on mine to route the cables through the existing cable tidying fabric stuff that comes with the ender 3, as that took me a real hour of tedious tweezer work to get through properly. However, I've used electrical tape to keep them neat, this will work fine.

![image](https://user-images.githubusercontent.com/26458780/75559639-b71e2d80-5a3b-11ea-91b5-9a5acf03eb92.png)

**Comment by ajlennon** [comment]
Great!

**Comment by MatthewCroughan** [comment]
I have however ran into a really peculiar issue with the sanguino_atmega1284p in this particular ender3, I am convinced it is broken in some way. 

Essentially it goes like this:
I am able to flash the bootloader to the chip via an Arduino Uno acting as an ISP programmer.
I am able to flash firmware to the chip *via* that same Arduino Uno acting as an ISP programmer.
I am *unable* to use the bootloader to flash firmware to the chip.

This is very annoying, this issue is not exhibited on `matt-ender3`. This should be okay as we do not often need to change the firmware on the device, though it is a real shame since it will mean that we have to take off the front panel and hook up dupont leads to the ISP headers inside of the unit each time we want to change the firmware, rather than using the USB on the front.

https://cdn.discordapp.com/attachments/336568402447564811/682967440929587220/JPEG_20200228_070616.jpg

![image](https://user-images.githubusercontent.com/26458780/75559986-53483480-5a3c-11ea-9532-e1601662b05c.png)

**Comment by MatthewCroughan** [comment]
Some of the dupont leads aren't making connections because of the fact that I used a crimper that was not fit for the gauge of cable used. We do not have a crimper in stock capable of crimping dupont leads and I have tried my hardest, but cannot get this to work due to that. Any idea where one might be, or can you lend me one @amcewen ?

**Comment by MatthewCroughan** [comment]
![image](https://user-images.githubusercontent.com/26458780/75685759-01e5b280-5c93-11ea-8beb-3129af2ff5dd.png)
For documentations sake, the BLtouch is wired accordingly.

**Comment by MatthewCroughan** [comment]
After massive trial and error with the ratcheting crimpers we have, I noticed that there is a raised portion on the crimper which is where you're supposed to put the back portion of the crimp that latches onto the wire. Doing this the other way around, despite this looking entirely innocent, will chop your wire off. The bed leveler is now installed. I have also routed the wires through the cable sheath, painstakingly.

![image](https://user-images.githubusercontent.com/26458780/75698917-73c7f700-5ca7-11ea-98ce-1ec76aca3f99.png)
